apps/mobile/src/navigation/TabsNavigator.js

```diff
@@ -2,18 +2,23 @@ import React from 'react'
 import { StyleSheet } from 'react-native'
 import { createBottomTabNavigator } from '@react-navigation/bottom-tabs'
 import { isIOS } from 'util/platform'
+import { get } from 'lodash/fp'
 import useCurrentUser from '@hylo/hooks/useCurrentUser'
+import { modalScreenName } from 'hooks/useIsModalScreen'
 import Icon from 'components/Icon'
 import {
   alabaster, black10OnCaribbeanGreen, gainsboro, gunsmoke, white
 } from 'style/colors'
 import HomeNavigator from 'navigation/HomeNavigator'
 import SearchNavigator from 'navigation/SearchNavigator'
 import MessagesNavigator from 'navigation/MessagesNavigator'
+import { useNavigation } from '@react-navigation/native'
```

Comment:
FYI - you probably have inferred by now that I have a preference for how import statements are ordered. My preference is basically as follows. I don't need in any action here, but you'll likely see me put shuffle things into this order whenever I touch a file that isn't already such:

1.	External Libraries – Core dependencies first (react, react-native), followed by utilities (lodash), data/state (redux, urql), and navigation.
2.	Project Files – Utilities, hooks, API queries/mutations, state management, and business logic. `@hylo` libraries at the end of this list. 
3.	Components & Styles – Components always come almost last, ideally imported in alphabetical order
4. `styles` / colors as the final imports if they exist.